include/deal.II/grid/tria_accessor.h

```diff
@@ -1296,6 +1296,30 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
   Point<spacedim> intermediate_point(const Point<structdim> &coordinates) const;
 
   /**
+   * This computes an approximate transformation from the real to the unit
+   * cell by inversion of an affine approximation of the $d$-linear function from the
+   * reference $d$-dimensional cell.
+   *
+   * The affine approximation of the unit to real cell approximation is found
+   * by a least squares fit of an affine function to the $2^d$ vertices of the
+   * present object. Due to construction, this transformation does never
+   * invert for any given point if the cell's geometry is valid. On the other
```

Comment:
I wanted to say that the affine transformation can always be inverted as long as the forward transformation makes sense. And it always does unless we have a mesh with cells of zero volume or other degenerate issues. I will try to reword.